cli/dcoscli/package/main.py

```diff
@@ -321,15 +283,15 @@ def _install(package_name, options_path, app_id, cli, app, yes):
 
     user_options = _user_options(options_path)
 
-    options = pkg.options(pkg_version, user_options)
+    options = pkg.options(pkg_revision, user_options)
 
-    if app and pkg.has_marathon_definition(pkg_version):
+    if app and pkg.has_marathon_definition(pkg_revision):
         # Install in Marathon
-        version_map = pkg.software_versions()
-        sw_version = version_map.get(pkg_version, '?')
+        revision_map = pkg.package_revisions_map()
+        package_version = revision_map.get(pkg_revision, '?')
```

Comment:
The map should always have key `pkg_revision`, since the `has_marathon_definition(pkg_revision)` is true